src/coprocessor/dag/expr/builtin_other.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::i64;
+
+use super::{EvalContext, Result, ScalarFunc};
+use coprocessor::codec::Datum;
+
+impl ScalarFunc {
+    #[inline]
+    pub fn bit_count(&self, ctx: &mut EvalContext, row: &[Datum]) -> Result<Option<i64>> {
+        let res = self.children[0].eval_int(ctx, row);
+        match res {
+            Ok(r) => if let Some(v) = r {
+                Ok(Some(i64::from(v.count_ones())))
+            } else {
+                Ok(None)
+            },
+            Err(e) => if e.is_overflow() {
+                Ok(Some(64i64))
```

Comment:
using 64 is enough here?

Reply:
@siddontang  this is corresponding to the implementation in [TiDB](https://github.com/pingcap/tidb/blob/master/expression/builtin_other.go)

Reply:
what I mean is `Ok(Some(64))`

Reply:
oh, yes, will refine here. :)